src/bin/head.rs

```diff
@@ -0,0 +1,137 @@
+#![deny(warnings)]
+
+extern crate coreutils;
+
+use std::env;
+use std::fs;
+use std::io::{self, BufRead, Read, Write};
+use coreutils::extra::{OptionalExt, fail};
+
+static MAN_PAGE: &'static str = r#"NAME
+    head - output the first part of a file
+
+SYNOPSIS
+    head [[-h | --help] | [-n LINES] | [-c BYTES]] [FILE ...]
+
+DESCRIPTION
+    Print the first 10 lines of each FILE to standard output. If there are no files, read the standard input. If there are multiple files, prefix each one with a header containing it's name.
+
+OPTIONS
+    -h
+    --help
+        Print this manual page.
+
+    -n [-]LINES
```

Comment:
Instead of adding `[-]`, you should note below that the integer is allowed to be negative.